rocksdb_admin/admin_handler.cpp

```diff
@@ -722,7 +725,7 @@ void AdminHandler::async_tm_addS3SstFilesToDB(
   rocksdb::IngestExternalFileOptions ifo;
   ifo.move_files = true;
   /* allow for overlapping keys */
```

Comment:
remove this comment now?

Reply:
isn't it useful to say what FLAGS_ rocksdb_allow_global_seqno does?